### DISCUSSION TITLE
To access parent/master dialog from child/detail dialog?

### DISCUSSION THREAD

**Discussion by erehtolleh** [serenity-is/Serenity#5369] [Help]
Does anyone know how to access the parent dialog from the child dialog using Serenity?  There is [Serenity Extra](https://github.com/dfaruque/Serenity.Extra) that gives you this function, but I rather not have to be dependent on it.

**Comment by VictorTomaili** [comment]
when open dialog, just add parameter or property on child dialog for parent.
